tensorflow/python/keras/backend_test.py

```diff
@@ -1736,5 +1736,34 @@ def test_batch_normalization(self):
     self.assertEqual(normed.shape.as_list(), [10, 3, 5, 5])
 
 
+class ControlOpsTests(test.TestCase):
+
+  def test_function_switch_basics(self):
```

Comment:
Can we test this under both graph and eager mode? eg add @test_util.run_all_in_graph_and_eager_modes to the test class?

Reply:
Sure, the same has been updated in the code.

Regards
Amit

Reply:
The annotation should be applied to the test class, which will cover all the test case within it.